### PR TITLE
 Add "Aliases" and "Decorators" to supported constructs section.

### PR DESCRIPTION
**Pull request by rchen152**
I'm not entirely clear on how type aliases containing type variables work, so I took my best guess at it.

### CODE REVIEW

File: pep-9999.rst [hunk 1]
```diff
@@ -87,6 +87,57 @@ Supported Constructs
...
+
+Type stubs may only use a fixed set of decorators:
+
+* ``classmethod``
+* ``staticmethod``
+* ``property`` (including ``.setter``)
+* ``abc.abstractmethod``
+* ``typing.overload``
+* ``asyncio.coroutines.coroutine``
+
```
Comment (JelleZijlstra) [inline pep-9999.rst:127]: Also `@typing.runtime_checkable`.

Comment (rchen152) [reply pep-9999.rst:127]: Thanks! This made me realize that I also forgot typing.type_check_only, so I tweaked this section to say that all ``typing`` decorators are okay.

File: pep-9999.rst [hunk 2]
```diff
@@ -87,6 +87,57 @@ Supported Constructs
 This sections lists constructs that compliant type checkers are expected
 to accept. Type stub authors can safely use these constructs.
 
+Aliases
+-------
+
+Type checkers should accept module-level and class-level aliases, e.g.::
+
+  IntList = List[int]
```
Comment (srittau) [inline pep-9999.rst]: `_IntList`? While not required, in practice this would be a private definition.

File: pep-9999.rst [hunk 3]
```diff
@@ -87,6 +87,57 @@ Supported Constructs
...
+must be substituted when the alias is used::
+
+  _K = TypeVar("_K")
+  _V = TypeVar("_V")
+  _MyMap = Dict[str, Dict[_K, _V]]
+
+  # either concrete types or other type variables can be substituted
+  def f(x: _MyMap[str, _V]) -> _V: ...
+  # explicitly substitute in Any rather than using a bare alias
+  def f(x: _MyMap[Any, Any]) -> Any: ...
```
Comment (srittau) [inline pep-9999.rst]: Since this is part of the same example, I'd use a different name for the second function.

**Review by srittau — APPROVED** [review]
LGTM, with two optional remarks. Especially the last example (substituting decorators) is very useful.

**Pull request merged**